launch/essentials/rgbd_to_ros.launch

```diff
@@ -25,6 +25,7 @@
 		<param name="link_threshold" value="0.8" />
 		<param name="probability_threshold" value="0.6" />
 		<param name="arm_norm_threshold" value="0.9" />
+		<param name="heuristic" value="head" />
```

Comment:
What does this do?

Reply:
Answer: break stuff...

Reply:
This does not seem to work together with the wave detection parameter as implemented in the people detector. What was used at RGO2018? Because this combination does not seem to work in practice. @Rayman can you shed some light here?